README.md

```diff
@@ -0,0 +1,314 @@
+
+[Linkedin Open Graph protocol documentation](https://www.linkedin.com/help/linkedin/answer/46687/making-your-website-shareable-on-linkedin).
+
+### OK (Odnoklassniki)
+```html
+<div class="odnoklassniki" data-imageurl="http://i.imgur.com/zunNbfY.jpg">Like</div>
+``` 
+* **url** - url to share.
+* **title** - text which is used as a title of created post.
+* **imageurl** - url to a picture which is used as a thumbnail for the post.
+* **counter** - if provided, blocks the API call and simply shows given value instead.
+
+[OK Open Graph protocol documentation](https://apiok.ru/en/ext/like).
+
+### Pinterest
+```html
+<div class="pinterest" data-media="https://placekitten.com/200/400">Pin</div>
+```
+* **url** - url to share.
+* **title** - text which is used as a comment to created pin.
+* **media** - URL of an image that overrides the image in the Pin Create form. 
+If not provided, Pinterest will try to find image at the given webpage. 
+Use the this attribute to provide a better-quality version of the image if you have one. 
+* **counter** - if provided, blocks the API call and simply shows given value instead.
+
+[Pinterest Open Graph protocol documentation](https://developers.pinterest.com/docs/rich-pins/overview/).
+
+### Reddit
+```html
+<div class="reddit">Submit</div>
+``` 
+Reddit counter is calculated as a sum score of the 5 most up-voted posts for a given link, across all sub-reddits.
+* **url** - url to share.
+* **title** - title of the post, defaults to the page title.
+* **counter** - if provided, blocks the API call and simply shows given value instead.
+
+[Reddit Open Graph protocol documentation](https://github.com/reddit-archive/reddit/blob/master/r2/r2/lib/media.py#L725).
+
+### Telegram
+```html
+<div class="telegram" data-title="Check this link above!">Send</div>
+```
+* **url** - url to share.
+* **title** - text that appears after the link in the shared message, defaults to the page title.
+
+[Telegram Open Graph protocol documentation](https://stackoverflow.com/questions/30160294).
+
+### Twitter
+```html
+<div class="twitter" data-via="ilyabirman" data-hashtags="kittens,puppies">Tweet</div>
+```
+* **url** - url to share.
+* **title** - comment that appears before the shared url.
+* **via** - indicates a specific user a source of the information. 
+Adds a clickable username to the tweet, like so: `My page: https://google.com/ via @ilyabirman`
+* **hashtags** - a comma-separated list of hashtags added to the tweet. Omit a preceding “#” from each hashtag.
+
+[Twitter Open Graph protocol documentation](https://developer.twitter.com/en/docs/twitter-for-websites/cards/guides/getting-started).
+
+### Viber
+```html
+<div class="viber">Send</div>
+```
+* **url** - url to share.
+* **title** - text that appears on a separate line after the shared url. 
+⚠ Viber share messages are not editable in the client application, 
+so if you don't wan't the title to appear, please set empty `data-title=""` attribute on the Viber button.
```

Comment:
doesn't work (as in, title is still used), most likely because of
https://github.com/NikolayRys/Likely/blob/changes-2-6/source/button.js#L95

Reply:
also nice catch, fixed too